text/1099-renderComponent.md

```diff
@@ -0,0 +1,403 @@
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+<!-- Replace "RFC title" with the title of your RFC -->
+
+# `renderComponent` 
+
+## Summary
+
+Add a new API to for rendering components into any DOM element.
+
+## Motivation
+
+Building an app from scratch requires a fair amount of boilerplate today, and `renderComponent` abstracts a minimal API that would allow easy integration of components in other enviroments, such as [Astro](https://astro.build/), or [Vitepress](https://vitepress.dev/), or [Storybook](https://storybook.js.org/), or runtime demos such as [JSBin](https://jsbin.com/).
+
+This also pairs well with [RFC#931: JS Representation of Template Tag](https://github.com/emberjs/rfcs/pull/931), in that both features together give folks enough to use Ember without compilation (compilation still recommended for optimizations, however).
+
+
+## Detailed design
+
+Users would import `renderComponent` from `@ember/renderer` (a pre-existing module)
+
+The interface:
+```ts
+/**
+ * Renders a component into an element, given a valid component definition.
+ */
+export function renderComponent(
+    /**
+     * The component definition to render.
+     *
+     * Any component that has had its manager registered is valid.
+     * For the component-types that ship with ember, manager registration 
+     * does not need to be worried about. 
+     */
+    component: object,
+    options: {
+        /**
+         * The element to render the component in to.
+         */
+        into: Element;
+
+        /**
+         * Optional owner. Defaults to `{}`, can be any object, but will need to implement the [Owner](https://api.emberjs.com/ember/release/classes/Owner) API for components within this render tree to access services.
+         */
+        owner?: object;
+
+        /**
+         * Optionally configure the rendering environment
+         */
+        env?: { 
+
+            /**
+             * When false, modifiers will not run.
+             */
+            isInteractive?: boolean; 
+
+            /**
+             * All other options are forwarded to the underlying renderer.
+             * (its API is currently private and out of scope for this RFC, 
+             *  so passing additional things here is also considered private API)
+             */
+            [rendererOption: string]?: unknown;
+        };
+
+        /**
+         * These args get passed to the rendered component
+         *
+         * If your args are reactive, re-rendering will happen automatically.
+         * 
+         */
+        args?: Record<string, unknown>;
+  }
+): RenderResult | undefined {
+    /* ... implementation details ... */
+}
+```
+
+`RenderResult` is a subset of the currently a pre-existing `interface` from `@glimmer/interfaces`, but would be exposed as public API via the only the return type of `renderComponent`.
```

Comment:
```suggestion
`RenderResult` is a subset of the currently pre-existing `interface` from `@glimmer/interfaces`, but would be exposed as public API via the only return type of `renderComponent`.
```

Reply:
Can you pr this change? As this rfc is already merged (and implemented!) i can't apply this suggestion